### ISSUE TITLE
Usage demo/example?

### ISSUE THREAD

**Issue by richardwestenra** [tumblbug/gatsby-source-workable#2]
Hi, is it possible to get a demo example that I can try out? I've been trying to follow the instructions in the readme as closely as possible, but I'm getting the following error in the node terminal:
> GraphQL Error Unknown field `allWorkableJob` on type `RootQueryType`

If you have any idea what's causing this issue, I'd be super grateful. Failing that, it would help me out a lot to have a working code example (like [the ones on the main Gatsby repo](https://github.com/gatsbyjs/gatsby/tree/master/examples)) that I can compare and reverse-engineer.

Thanks!

**Comment by richardwestenra** [comment]
Note: I've figured out what was causing the issue (a bug in package.json that should be fixed by PR #3), but it would still be nice to get a wee example to play with.

**Comment by bjrnt** [comment]
I agree - an example would be great! The only problem is that Workable is a paid service, so it might be difficult to provide a cloneable repo.

**Comment by richardwestenra** [comment]
Agreed.

I raised a similar issue on the [main Gatsby repo](https://github.com/gatsbyjs/gatsby/issues/2413), and was thinking that I might set up a version with either the API keys in dotfiles, so that they're not exposed, or else leave them blank in the config, e.g. something like `apiKey: 'WORKABLE_API_KEY_GOES_HERE'.

Then I could just set up a closed-source demo elsewhere.
